PendingReleaseNotes

```diff
@@ -1,5 +1,8 @@
 >=17.0.0
 
+* Filestore will be deprecated in Quincy, considering BlueStore has been
+  the default objectstore for quite some time.
```

Comment:
The note can be augmented indicating that 'wpq' scheduler is employed for filestore OSDs.